BurnOutSharp/PackerType/IntelInstallationFramework.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+namespace BurnOutSharp.PackerType
+{
+    // TODO: Add extraction, seems to primarily use MSZip compression.
+    public class IntelInstallationFramework : IContentCheck
+    {
+        /// <inheritdoc/>
+        public bool ShouldScan(byte[] magic) => true;
+
+        /// <inheritdoc/>
+        public string CheckContents(string file, byte[] fileContent, bool includePosition = false)
+        {
+            var fileDetails = Utilities.GetFileVersionInfo(file);
+
+            string name = fileDetails?.FileDescription.Trim();
```

Comment:
Try to better organize this section by field in the `FileVersionInfo`. If that's null, you can also just return right away instead of needing the `?.ProductName` and `?.FileDescription` checks.